crates/bevy_core_widgets/src/core_scrollbar.rs

```diff
@@ -0,0 +1,310 @@
+use bevy_app::{App, Plugin, PostUpdate};
+use bevy_ecs::{
+    component::Component,
+    entity::Entity,
+    hierarchy::{ChildOf, Children},
+    observer::On,
+    query::{With, Without},
+    system::{Query, Res},
+};
+use bevy_math::Vec2;
+use bevy_picking::events::{Drag, DragEnd, DragStart, Pointer, Press};
+use bevy_ui::{
+    ComputedNode, ComputedNodeTarget, Node, ScrollPosition, UiGlobalTransform, UiScale, Val,
+};
+
+/// Used to select the orientation of the scrollbar.
+#[derive(Debug, Default, Clone, Copy, PartialEq)]
+pub enum Orientation {
```

Comment:
This name is too general for the scope; it's going to mess up tab-imports. We should rename this to `ScrollbarOrientation`.

Reply:
Went with `ControlOrientation`.